src/pe_source/cybersixgill.py

```diff
@@ -0,0 +1,407 @@
+"""Collect Cybersixgill data."""
+
+# Standard Python Libraries
+from datetime import date, datetime, timedelta
+import logging
+import sys
+import traceback
+
+from .data.pe_db.db_query_source import (
+    get_breaches,
+    get_data_source_uid,
+    get_orgs,
+    insert_sixgill_alerts,
+    insert_sixgill_breaches,
+    insert_sixgill_credentials,
+    insert_sixgill_mentions,
+    insert_sixgill_topCVEs,
+)
+from .data.sixgill.api import get_sixgill_organizations
+from .data.sixgill.source import (
+    alerts,
+    alias_organization,
+    all_assets_list,
+    creds,
+    cve_summary,
+    get_alerts_content,
+    mentions,
+    root_domains,
+    top_cves,
+)
+
+# Set todays date formatted YYYY-MM-DD and the start_date 30 days prior
+TODAY = date.today()
+DAYS_BACK = timedelta(days=30)
+MENTIONS_DAYS_BACK = timedelta(days=16)
+MENTIONS_START_DATE = str(TODAY - MENTIONS_DAYS_BACK)
+END_DATE = str(TODAY)
+DATE_SPAN = f"[{MENTIONS_START_DATE} TO {END_DATE}]"
+
+# Set dates to YYYY-MM-DD H:M:S format
+NOW = datetime.now()
+START_DATE_TIME = (NOW - DAYS_BACK).strftime("%Y-%m-%d %H:%M:%S")
+END_DATE_TIME = NOW.strftime("%Y-%m-%d %H:%M:%S")
+
+LOGGER = logging.getLogger(__name__)
+
+
+class Cybersixgill:
+    """Fetch Cybersixgill data."""
+
+    def __init__(self, orgs_list, method_list, soc_med_included):
+        """Initialize Cybersixgill class."""
+        self.orgs_list = orgs_list
+        self.method_list = method_list
+        self.soc_med_included = soc_med_included
+
+    def run_cybersixgill(self):
+        """Run Cybersixgill api calls."""
+        orgs_list = self.orgs_list
+        method_list = self.method_list
+        soc_med_included = self.soc_med_included
+
+        # Get org info from PE database
+        pe_orgs = get_orgs()
+        # Get Cybersixgill org info
+        sixgill_orgs = get_sixgill_organizations()
+        failed = []
+        count = 0
+
+        # Get data source uid
+        source_uid = get_data_source_uid("Cybersixgill")
+
+        # Run top CVEs. Same for all orgs
+        if "topCVEs" in method_list:
+            if self.get_topCVEs(source_uid) == 1:
+                failed.append("Top CVEs")
+
+        list = ""
+        for pe_org in pe_orgs:
+            list = list + pe_org["cyhy_db_name"] + ","
+        print(list)
```

Comment:
This output should be logged as either debug or info, rather than printed.

Also, you can get rid of that temporary `list` variable with something like this:
```suggestion
        LOGGER.info(",".join([org["cyhy_db_name"] for org in pe_orgs]))
```